apps/public-docsite-v9/src/Concepts/Slots/Slots.stories.mdx

```diff
@@ -0,0 +1,365 @@
+import { Meta, Story } from '@storybook/addon-docs';
+import { SlotSketch } from './SlotSketch.stories';
+
+<Meta title="Concepts/Developer/Customizing Components with Slots" />
+
+## Customizing Components with Slots
+
+Fluent UI React components have parts that are designed to be replaced.
+These are called slots.
```

Comment:
Does this render as a new line in our docs? The markdown preview shows it as just inline text.